junit-jupiter-api/src/main/java/org/junit/jupiter/api/MethodOrderer.java

```diff
@@ -104,15 +105,35 @@
 		return Optional.of(ExecutionMode.SAME_THREAD);
 	}
 
+	/**
+	 * {@code MethodOrderer} that sorts methods alphanumerically based on their
+	 * names using {@link String#compareTo(String)}.
+	 *
+	 * <p>If two methods have the same name, {@code String} representations of
+	 * their formal parameter lists will be used as a fallback for comparing the
+	 * methods.</p>
+	 * <p> <b>This class has been deprecated in favor of {@code MethodOrderer.MethodName} and will be
+	 * removed in 6.0</b></p>
+	 *
+	 * @since 5.4
+	 * @deprecated Deprecated in favor of MethodOrderer.MethodName. Will be removed in 6.0
+	 */
+	@API(status = DEPRECATED, since = "5.4")
```

Comment:
I misunderstood the `API.since` value to describe since when the API existed as is the `since` in javadoc. While this describes status changes. 
Thanks for pointing it out!